### PR TITLE
Bump Shark-TestSuite hash to head

### PR DESCRIPTION
**Pull request by rsuderman**
Update SHARK-Turbine to https://github.com/nod-ai/SHARK-TestSuite/compare/1da89a12a8b00dc77506d702f61300803b6240b7...f5615ab29da491c0047146258dfa3a0c40c735e5

Faster file downloads: https://github.com/nod-ai/SHARK-TestSuite/commit/6c1ff8f361addcf85442fb2021dce3e27d35a6bf
Import more ONNX tests cases / correct some element types: https://github.com/nod-ai/SHARK-TestSuite/commit/f5615ab29da491c0047146258dfa3a0c40c735e5

### CODE REVIEW

File: .github/workflows/pkgci_regression_test.yml [hunk 1]
```diff
@@ -89,7 +89,7 @@ jobs:
         uses: actions/checkout@v4.1.7
         with:
           repository: nod-ai/SHARK-TestSuite
-          ref: 1da89a12a8b00dc77506d702f61300803b6240b7
+          ref: f5615ab29da491c0047146258dfa3a0c40c735e5
```
Comment (ScottTodd) [inline .github/workflows/pkgci_regression_test.yml:92]: Please also update the hash in the `test_models` job below, so there is a single value to find/replace across the whole repo.

**Review by ScottTodd — COMMENTED** [review]
Can you add a summary of what changed to the PR title? Since this isn't using a submodule, you could also include this link so people can view the commits in GitHub's UI: https://github.com/nod-ai/SHARK-TestSuite/compare/1da89a12a8b00dc77506d702f61300803b6240b7...f5615ab29da491c0047146258dfa3a0c40c735e5

Looks like the main changes affecting this repo are:
* Faster file downloads: https://github.com/nod-ai/SHARK-TestSuite/commit/6c1ff8f361addcf85442fb2021dce3e27d35a6bf
* Import more ONNX tests cases / correct some element types: https://github.com/nod-ai/SHARK-TestSuite/commit/f5615ab29da491c0047146258dfa3a0c40c735e5

File: .github/workflows/pkgci_regression_test.yml (hunk 1 above)
Comment (rsuderman) [reply .github/workflows/pkgci_regression_test.yml:92]: Done.

**Comment by ScottTodd** [comment]
Just waiting on test results / XFAIL updates then LGTM.

**Review by ScottTodd — APPROVED** [review]

**Pull request merged**